src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const mainImage = document.querySelector('.gallery__large-img');
+const thumbs = document.querySelector('#thumbs');
+
+thumbs.addEventListener('click', (mouseEvent) => {
```

Comment:
```suggestion
thumbs.addEventListener('click', (event) => {
```

better to use common and conventional name 'event' or 'e'